R/vdiagram-render.R

```diff
@@ -181,20 +135,20 @@ addRatingShifts <- function(vplot, vdiagramData, styles) {
 vdiagramTable <- function(reportObject){
   ratingShifts <- fetchRatingShifts(reportObject)
   
-  shiftPoints <- ratingShifts$shiftPoints
+  shiftPoints <- ratingShifts[["shiftPoints"]]
```

Comment:
Some of this is repeated in `parseVDiagramData`, but I don't see an easy way to share functionality. I'd suggest this be two different functions:

1. `fetchVDiagramTable <- function(ratingShifts)` which returns a list of data, where `names(data)` is `shiftPoints, stagePoints, shiftID, startTime, rating, nShift`
2. `renderVDiagramTable <- function(shiftDataList)`